package/gluon-core/luasrc/lib/gluon/upgrade/200-wireless

```diff
@@ -179,6 +175,22 @@ local function configure_mesh_wireless(radio, index, config, disabled)
 	)
 end
 
+local function set_channels(radio, radio_name, config)
+	if wireless.preserve_channels(uci) then
+		return
+	end
+	local channel = get_channel(radio, config)
+	uci:set('wireless', radio_name, 'channel', channel)
+
+	if radio.band == '5g' then
```

Comment:
I think for consistency it would make sense to use the same condition `radio.band == '5g' and is_outdoor()` as in `get_channel()` (where the `is_outdoor()` check is right now, so the `uci:set(..., 'channels', chanlist)` call is also done for 2g, always setting it to nil)